storybookjs/storybook
# Addon-docs: Add CSS Shadow Parts to web-component props table

Issue: #10353 

## What I did

Renamed the `css` to `cssCustomProperties` of the props table for the framework web-components in addon-docs and added `cssShadowParts` to the same table

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes
- Does this need a new example in the kitchen sink apps? Yes
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
